app/executor.php

```diff
@@ -412,23 +410,18 @@ function execute(string $trigger, string $projectId, string $executionId, string
                 '$id' => $buildId,
                 '$read' => ($userId !== '') ? ['user:' . $userId] : [],
                 '$write' => [],
-                'dateCreated' => time(),
+                'startTime' => time(),
+                'deploymentId' => $deployment->getId(),
                 'status' => 'processing',
                 'outputPath' => '',
                 'runtime' => $function->getAttribute('runtime', ''),
                 'source' => $deployment->getAttribute('path'),
                 'sourceType' => Storage::DEVICE_LOCAL,
                 'stdout' => '',
                 'stderr' => '',
-                'time' => 0,
-                'vars' => [
-                    'ENTRYPOINT_NAME' => $deployment->getAttribute('entrypoint'),
-                    'APPWRITE_FUNCTION_ID' => $function->getId(),
-                    'APPWRITE_FUNCTION_NAME' => $function->getAttribute('name', ''),
-                    'APPWRITE_FUNCTION_RUNTIME_NAME' => $runtime['name'],
-                    'APPWRITE_FUNCTION_RUNTIME_VERSION' => $runtime['version'],
-                    'APPWRITE_FUNCTION_PROJECT_ID' => $projectId,
-                ]
+                'endTime' => 0,
+                'duration' => 0,
+                'search' => implode(' ', [$deployment->getId(), $buildId]),
```

Comment:
I don't think we need the `search` field since we don't use it anywhere. 

Let's remove it from the collection as well.